cmd/restic/global.go

```diff
@@ -382,15 +384,42 @@ func ReadPasswordTwice(gopts GlobalOptions, prompt1, prompt2 string) (string, er
 	return pw1, nil
 }
 
+func ReadRepo(opts GlobalOptions) (string, error) {
+	if opts.Repo == "" && opts.RepositoryFile == "" {
+		return "", errors.Fatal("Please specify repository location (-r or --repository-file)")
+	}
+
+	repo := opts.Repo
+	if opts.RepositoryFile != "" {
```

Comment:
Please add an integration test which tests reading the repository path from a file. (e.g. init a repository using a path stored in `Repo` then write the path to a RepositoryFile and try to access the repository again.)